cluster-autoscaler/cloudprovider/gce/templates.go

```diff
@@ -352,19 +372,55 @@ func extractTaintsFromKubeEnv(kubeEnv string) ([]apiv1.Taint, error) {
 }
 
 func extractKubeReservedFromKubeEnv(kubeEnv string) (string, error) {
-	kubeletArgs, err := extractFromKubeEnv(kubeEnv, "KUBELET_TEST_ARGS")
+	// In v1.10+, kube-reserved is only exposed for the autoscaler via AUTOSCALER_ENV_VARS
+	// see kubernetes/kubernetes#61119. We try AUTOSCALER_ENV_VARS first, then
+	// fall back to the old way.
+	kubeReserved, err := extractAutoscalerVarFromKubeEnv(kubeEnv, "kube_reserved")
```

Comment:
In https://github.com/kubernetes/kubernetes/pull/61119 you have a comment that kube-reserved will need to be pulled from KUBELET_ARGS but this code only checks AUTOSCALER_ARGS with a fallback to KUBELET_TEST_ARGS so it won't find the parameter for 1.10 nodes.

Reply:
Users can just edit their MIG templates to include it in AUTOSCALER_ENV_VARS on GCE if they need it. We can probably just remove the comment. I'd recommend we have a release note in the next autoscaler version to that effect.

Reply:
Good idea, I can take care of adding to release notes.